documentation.json

```diff
@@ -0,0 +1,128 @@
+[
+  {
+    "path": "/",
+    "methods": [
+      "GET"
+    ],
+    "middlewares": [
+      "anonymous"
+    ]
+  },
+  {
+    "path": "/movies",
+    "methods": [
+      "GET"
+    ],
+    "middlewares": [
+      "anonymous"
+    ]
+  },
```

Comment:
To be more descriptive in your api usage, you could add the queries to filter the data:

```json
 "middlewares": [
    "anonymous"
   ],
  "query_params": {
    "category": "Filter movies by category",
    "year": "Filter movies by year",
    "win": "Filter movies by win status (true/false)",
    "nominee": "Filter movies by nominee",
    "film": "Filter movies by film or series name"
  }
```

Reply:
will do!